src/Util/TestDox/TestDoxPrinter.php

```diff
@@ -96,6 +128,11 @@ public function setOriginalExecutionOrder(array $order): void
         $this->enableOutputBuffer     = !empty($order);
     }
 
+    public function setEnableOutputBuffer(bool $enabled = true): void
```

Comment:
Since the default is `false`, perhaps we only need a method `enableOutputBuffer()`, and maybe `disableOutputBuffer()`, both without parameters?

Reply:
Good point. There was some obscure reason I hadn't done that yet. Will check my notes.